spock-core/src/main/java/org/spockframework/runtime/Sputnik.java

```diff
@@ -51,6 +67,27 @@ public Sputnik(Class<?> clazz) throws InitializationError {
     this.clazz = clazz;
   }
 
+  @Override
+  protected List getChildren() {
+    throw new UnsupportedOperationException("workflow from ParentRunner not supported");
```

Comment:
Are these three UnsupportedOperationEx unproblematic? 
Why can't those be implemented? We have access to the SpecInfo.